src/sql/pg/create.sql

```diff
@@ -0,0 +1,147 @@
+--
+-- PostgreSQL
+--
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SET check_function_bodies = false;
+SET client_min_messages = "warning";
+SET search_path = "public", "pg_catalog";
+--
+-- Sequences
+--
+
+CREATE SEQUENCE "N_CASES_ID_SEQ"
+  INCREMENT 1 
+  START 1
+  CACHE 100;
+
+CREATE SEQUENCE "N_NOTIFICATION_ID_SEQ"
+  INCREMENT 1 
+  START 1
+  CACHE 100;
+
+--
+-- Tables
+--
+CREATE TABLE "N_LU_CASE_STATUS"
+(
+    "CASE_STATUS_ID"             INTEGER NOT NULL ,
+    "CASE_STATUS_CODE"           CHARACTER VARYING(20) NOT NULL ,
+    "CASE_STATUS_DESC"           CHARACTER VARYING(255) NULL ,
+    "DB_CREATED_DTTM"            TIMESTAMP WITH TIME ZONE DEFAULT  NOW()  NOT NULL ,
+    "DB_UPDATED_DTTM"            TIMESTAMP WITH TIME ZONE DEFAULT  NOW()  NOT NULL ,
+    "DB_UPDATED_USER"            CHARACTER VARYING(40) DEFAULT  CURRENT_USER  NOT NULL 
+);
+
+CREATE INDEX "N_XPK_LU_CASE_STATE" ON "N_LU_CASE_STATUS"
+("CASE_STATUS_ID"   ASC);
+
+ALTER TABLE "N_LU_CASE_STATUS" ADD CONSTRAINT "N_PK_LU_CASE_STATE" PRIMARY KEY ("CASE_STATUS_ID");
+
+CREATE UNIQUE INDEX "N_XAK1_LU_CASE_STATE" ON "N_LU_CASE_STATUS"
+("CASE_STATUS_CODE"   ASC);
+
+CREATE TABLE "N_CASES"
+(
+    "CASE_ID"                    INTEGER NOT NULL DEFAULT nextval('"N_CASES_ID_SEQ"'::regclass) ,
+    "CASE_UUID"                  CHARACTER VARYING(64) NOT NULL ,
+    "CASE_STATUS_ID"             INTEGER NOT NULL REFERENCES "N_LU_CASE_STATUS"("CASE_STATUS_ID") ,
+    "EXECUTION_CONDITION_TYPE"   CHARACTER VARYING(20) NULL ,
+    "EXECUTION_CONDITION_DIGEST" CHARACTER VARYING(4000) NULL ,
+    "EXEC_COND_FULFILLMENT"      CHARACTER VARYING(4000) NULL ,
+    "EXPIRES_DTTM"               TIMESTAMP WITH TIME ZONE NULL ,
+    "NOTARIES"                   CHARACTER VARYING(4000) NULL ,
+    "NOTIFICATION_TARGETS"       CHARACTER VARYING(4000) NULL ,
+    "DB_CREATED_DTTM"            TIMESTAMP WITH TIME ZONE DEFAULT  NOW()  NOT NULL ,
+    "DB_UPDATED_DTTM"            TIMESTAMP WITH TIME ZONE DEFAULT  NOW()  NOT NULL ,
+    "DB_UPDATED_USER"            CHARACTER VARYING(40) DEFAULT  CURRENT_USER  NOT NULL 
+);
+
+CREATE INDEX "N_XPK_CASES" ON "N_CASES"
+("CASE_ID"   DESC);
+
+ALTER TABLE ONLY "N_CASES"
+    ADD CONSTRAINT "N_PK_CASES" PRIMARY KEY ("CASE_ID");
+
+CREATE UNIQUE INDEX "N_XAK1CASES_GUID" ON "N_CASES"
```

Comment:
I think one of the rules in the email was that we can drop the enumerations, so for example this would be `N_XAK_CASES_GUID`